stable/opa/values.yaml

```diff
@@ -45,7 +45,23 @@ admissionControllerKey: ""
 
 # Docker image and tag to deploy.
 image: openpolicyagent/opa
-imageTag: 0.10.1
+imageTag: 0.10.2
+
+mgmt:
+  enabled: true
```

Comment:
kube-mgmt and OPA's bundle feature are not compatible today. If you enable both, the bundles will overwrite the policies/data loaded by kube-mgmt.

If someone enables this and has bundles enabled above, they're going to get unexpected behavior.

I'd rather have bundles on by default because they're simpler and the preferred long-term approach for policy/data distribution.

Can we have the helm chart validate that kube-mgmt and bundles are not both enabled at the same time?

Reply:
the policy configmap replication is disabled by default. So kube-mgmt can be used for replication of other objects with bundles. I think that's ok?